common/lib/container-definitions/src/runtime.ts

```diff
@@ -161,6 +161,12 @@ export interface IContainerContext extends IDisposable {
     getLoadedFromVersion(): IVersion | undefined;
 
     updateDirtyContainerState(dirty: boolean): void;
+    /**
+     * WARNING: this id is meant for telemetry usages ONLY, not recommended for other consumption
+     * This id is not supposed to be exposed anywhere else. It is not the same with document id as this is
```

Comment:
I'd probably remove verbiage about "document id" as we do not define this term anywhere in runtime (other than when using telemetry where we have docId property taht simply logs this id :)).